back/engines/commercial/content_builder/app/models/content_builder/layout_image.rb

```diff
@@ -12,12 +12,14 @@
 #
 module ContentBuilder
   class LayoutImage < ApplicationRecord
-    mount_base64_uploader :image, LayoutImageUploader
+    attr_accessor :skip_image_presence
```

Comment:
Changes to this file add in the new validation of presence of `:image`, which can be skipped, so that we can skip this when applying templates (as otherwise the validation will cause the template application to fail. For more details see #3902)